Script/AtomicEditor/editor/Editor.ts

```diff
@@ -21,19 +21,18 @@
 //
 
 import EditorUI = require("ui/EditorUI");
-import UIEvents = require("ui/UIEvents");
 import PlayMode = require("ui/playmode/PlayMode");
 import EditorLicense = require("./EditorLicense");
 import EditorEvents = require("./EditorEvents");
 import Preferences = require("./Preferences");
 
-class Editor extends Atomic.ScriptObject {
+class AtomicEditor extends Atomic.ScriptObject {
 
```

Comment:
This was actually a forced change.  I was getting name collisions trying to reference both the Editor class in the project and the Editor namespace exposed from cpp. :)